t/maxconns.t

```diff
@@ -11,22 +11,25 @@ use FindBin qw($Bin);
 use lib "$Bin/lib";
 use MemcachedTest;
 
-plan skip_all => "maxconns test does not work";
-exit 0;
-
-plan tests => 6;
-
-# start up a server with 10 maximum connections
-my $server = new_memcached('-c 100');
-my $sock = $server->sock;
-my @sockets;
-
-ok(defined($sock), 'Connection 0');
-push (@sockets, $sock);
-
-
-foreach my $conn (1..10) {
-  $sock = $server->new_sock;
-  ok(defined($sock), "Made connection $conn");
-  push(@sockets, $sock);
+my $server = new_memcached('-c 30','','true');
+
+my $stat_sock = $server->sock;
+my @sockets = ();
+my $num_sockets;
+my $rejected_conns = 0;
+for (1 .. 5) {
+  my $sock = $server->new_sock;
+  if (defined($sock)) {
+    push (@sockets, $sock);
+  } else {
+    $rejected_conns = $rejected_conns + 1;
```

Comment:
hmm is this actually right? You should be able to read the error message from the socket and validate you're getting it.

Reply:
fixed the test now